app/lib/pre_assembly/digital_object.rb

```diff
@@ -73,7 +73,7 @@ def pre_assemble
                  message: "can't be opened for a new version; cannot re-accession when version > 1 unless object can be opened" }
       end
 
-      @assembly_directory = AssemblyDirectory.create(druid_id: druid.id)
+      @assembly_directory = AssemblyDirectory.create(druid_id: druid.id, common_path: common_path)
       stage_files
```

Comment:
pass in the common path to all staged files so we can remove it when indicating the path to any individual staged file